workbase/src/renderer/components/Visualiser/ContextMenuUtils.js

```diff
@@ -0,0 +1,27 @@
+import dataVisualiserStore from '@/components/Visualiser/store';
```

Comment:
instead of importing the store, it's probably better to pass the `state` as a param to the methods:

`function verifyEnableExplain( state ){ }`

it's also easier to test (you can pass a mock object to the functions instead of a real state)